src/Redux/api.tsx

```diff
@@ -807,10 +812,12 @@ const routes = {
   flagInventoryItem: {
     path: "/api/v1/facility/{facility_external_id}/inventory/{external_id}/flag/",
     method: "PUT",
+    TRes: Type<PaginatedResponse<InventoryLogResponse>>(),
   },
   deleteLastInventoryLog: {
     path: "/api/v1/facility/{facility_external_id}/inventory/delete_last/?item={id}",
```

Comment:
remove the query params from the route path. And pass it as query params in the request that invokes this.

Reply:
@rithviknishad 
`facility_external_id` and  `external_id` are given as pathParams in the below image. Is it possible to pass it from the request?
<img width="928" alt="image" src="https://github.com/coronasafe/care_fe/assets/114745442/6c6deb74-121b-4adb-a0cb-89ce711e5e38">

Reply:
facility_external_id and external_id are path params. That need not be removed. Remove the query param: `?item={id}` and pass it as `{ query: { item: id }}` in the request/useQuery